docs/index.md

```diff
@@ -14,9 +14,9 @@ og_image: "/images/plugins/turbot/code-social-graphic.png"
 
 Source code can be any string or data for querying.
 
-[Steampipe](https://steampipe.io) is an open source CLI to instantly query cloud APIs using SQL.
+[Steampipe](https://steampipe.io) is an open source CLI to instantly query local data or APIs (cloud or not) using SQL.
```

Comment:
Thanks @pdecat for making the changes, however, we have been using this description across all our plugins(agreed we have gone beyond just cloud APIs).

@cbruno10 do you think we should update all our plugins to use the following description - 
```
[Steampipe](https://steampipe.io) is an open source CLI to instantly query local data or APIs (cloud or not) using SQL.
```

Reply:
@misraved Do we have any other messaging on steampipe.io, or in other marketing material, that describes Steampipe?

Reply:
@cbruno10 currently we don't have any marketing material that describes Steampipe. 

This [introductory blog](https://steampipe.io/blog/introducing-steampipe) only talks about using Steampipe for Cloud APIs though.